css/selectors/active.json

```diff
@@ -5,6 +5,10 @@
         "__compat": {
           "description": "<code>:active</code>",
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/:active",
+          "spec_url": [
+            "https://html.spec.whatwg.org/multipage/scripting.html#selector-active",
+            "https://drafts.csswg.org/selectors/#active-pseudo"
```

Comment:
https://drafts.csswg.org/selectors/#the-active-pseudo is the section, if we're still doing that.

Reply:
> [drafts.csswg.org/selectors/#the-active-pseudo](https://drafts.csswg.org/selectors/#the-active-pseudo) is the section, if we're still doing that.

Yeah I guess we need to decide. But for all features in this patch, what’s currently here was just automatically scraped out from the Specification table in the MDN article for the feature.

Reply:
@Elchi3 did we decide? :)

Reply:
I think we decided for the section as it is better UX. I haven't heard any other argument, so I was fine with deciding on sections.